scripts/fix.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-export PATH=env/bin:${PATH}
+export PATH=$(poetry show -v | grep virtualenv | sed 's/Using virtualenv: //')/bin:${PATH}
 
 black setup.py zoloto tests benchmarks examples
```

Comment:
Need to remove `setup.py` from here, too.

Reply:
ba3bc9e